pkg/auth/jwt_test.go

```diff
@@ -93,8 +93,11 @@ const dexkeys = `{
   ]
 }`
 
+const sharedSecretKey = `ZXhhbXBsZS1hcHAtc2VjcmV0`
+
 // generated from running onos-gui against the Dex IDP - expires 12 Jul 2030
 const sampleTokenOnosGuiAndDex = `eyJhbGciOiJSUzI1NiIsImtpZCI6ImRmMzYyZGMzNzRjNWJmMWEyZjEwMzllZDhkNjYyOWM5MGI5M2EwNGUifQ.eyJpc3MiOiJodHRwOi8vZGV4OjMyMDAwIiwic3ViIjoiQ2lRd09HRTROamcwWWkxa1lqZzRMVFJpTnpNdE9UQmhPUzB6WTJReE5qWXhaalUwTmpnU0JXeHZZMkZzIiwiYXVkIjoib25vcy1ndWkiLCJleHAiOjE5MDk5Mzg0NDcsImlhdCI6MTU5NDU3ODQ0Nywibm9uY2UiOiJjelJ2TmtoTFVYbFhVSE5EYWpsMmNHSlFjMVZOUld0T2NERmtMbkJ0ZHkxbFJTMVlkM0YzVUVGNE5IZFEiLCJhdF9oYXNoIjoiWXcxQ1M1UmdvYVRVWUNUZWtxUkxBZyIsImVtYWlsIjoic2VhbkBvcGVubmV0d29ya2luZy5vcmciLCJlbWFpbF92ZXJpZmllZCI6dHJ1ZSwibmFtZSI6InNlYW4ifQ.OMxzPd9CD7CoZVPRYSXJdLy8QCQjkIyjD7xFKBbDDsOCvIK9MkNHBTgQfrc3gmv8DZaAyzj9abLRS1TPWBwfQl5QJW9unXbjtk4KukXxv0CbKN3NyRV5Nm5YlGm66DIlPMj8udlqyau_xUJXrVC4T13sPo1CVpnAHut6Rr9zwPj3pVLwPXO2dkqHH4c1YM9Lyg8fpMv5eGd3iN6xcRMCcxUFkqffvwS1mCW6BUoBvnbMMZEaNokAC6HcWD8EB_m-Z7nt_xP_C_mnnZGFJNGDU0fRUjdGRxQ6AHYVg1zCS_B8P1IxkIe6tnBRi8309s99Q4MhlDWSoju_fe8pU7iLwQ`
+const sampleTokenHS256Signature = `eyJhbGciOiJIUzI1NiIsImtpZCI6ImRmMzYyZGMzNzRjNWJmMWEyZjEwMzllZDhkNjYyOWM5MGI5M2EwNGUifQ.eyJpc3MiOiJodHRwOi8vZGV4OjMyMDAwIiwic3ViIjoiQ2lRd09HRTROamcwWWkxa1lqZzRMVFJpTnpNdE9UQmhPUzB6WTJReE5qWXhaalUwTmpnU0JXeHZZMkZzIiwiYXVkIjoib25vcy1ndWkiLCJleHAiOjE5MDk5Mzg0NDcsImlhdCI6MTU5NDU3ODQ0Nywibm9uY2UiOiJjelJ2TmtoTFVYbFhVSE5EYWpsMmNHSlFjMVZOUld0T2NERmtMbkJ0ZHkxbFJTMVlkM0YzVUVGNE5IZFEiLCJhdF9oYXNoIjoiWXcxQ1M1UmdvYVRVWUNUZWtxUkxBZyIsImVtYWlsIjoic2VhbkBvcGVubmV0d29ya2luZy5vcmciLCJlbWFpbF92ZXJpZmllZCI6dHJ1ZSwibmFtZSI6InNlYW4ifQ.vAx1WCxsJNEvpfMvTU0IqnWgen1xmS3lOjkorZL5uWU`
```

Comment:
I don't  see why a second one is required

Reply:
The signature is different because the algorithm which is used is HS256

Reply:
For testing of this part of the code: https://github.com/onosproject/onos-lib-go/blob/master/pkg/auth/jwt.go#L62